src/subscription/azext_subscription/commands.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import CliCommandType
+from ._client_factory import subscription_definitions_mgmt_client_factory
+from ._exception_handler import subscription_exception_handler
+
+
+def load_command_table(self, _):
+    subscription_definition_util = CliCommandType(
+        operations_tmpl='azext_subscription.subscription.operations.subscription_definitions_operations#SubscriptionDefinitionsOperations.{}',
+        client_factory=subscription_definitions_mgmt_client_factory,
+        client_arg_name='self',
```

Comment:
`client_arg_name` shouldn't be needed. What happens if you omit it?

Reply:
It may be due to https://github.com/Azure/azure-cli/commit/6808e6b023721c49c8661d5d55b1de3ab97bf25e#diff-103131eb32c8ff55661bc902248c80a6R322?

Reply:
Yes, my commands don't work without this ("missing parameter: self"). This was one of the main surprises I hit when converting from module to extension.

Reply:
@derekbekoe I see. Can you open an issue in the main CLI repo so I look into this in an upcoming sprint? Ideally this shouldn't be needed.